runner/runner.go

```diff
@@ -11,23 +11,23 @@ import (
 )
 
 type (
-	ConfigBody    = config.Body
-	ConfigGlobal  = config.Global
-	ConfigRequest = config.Request
-	ConfigRunner  = config.Runner
-	ConfigOutput  = config.Output
+	Config         = config.Global
+	RequestConfig  = config.Request
+	RequestBody    = config.RequestBody
+	RecorderConfig = config.Runner
+	OutputConfig   = config.Output
```

Comment:
Is is ok to not reference it by its domain name, "Report" (if i get it correctly) ?

Reply:
`Output` does not quite refer to the notion of `Report`, but rather refers to anything related to the CLI output in a user's point of view: `silent`, `template`.

My guess is we should not have any notion of output in the runner (we already removed a lot, I think it remains only in `config`) because it's specific to the CLI. However we cannot do that until we decorrelate the testing part (`template`) from the output.

Reply:
Totally agree! Do we need some comments in code or an issue to track this?

Reply:
Definitely 😛 
Done: https://github.com/benchttp/engine/issues/15